MariaCraciun1_Task3/SConstruct

```diff
@@ -0,0 +1,8 @@
+from SCons.Environment import Environment
+
+env = Environment()   
+env.Append(CCFLAGS = ['-g'])
+env.Append(LIBS=['librt', 'pthread'])
+
+env.Program(target = "server", source = ["server.cpp", "DynamicStringArray.cpp"], LIBS= ["librt", "pthread"])
+env.Program(target = "client", source = ["client.cpp", "DynamicStringArray.o"], LIBS= ["librt", "pthread"])
```

Comment:
You don't need DynamicStringArray in the client.

Reply:
This change will be visible with the push for the implementation of help command.